testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -1523,7 +1522,7 @@ public void testMultiSearch() throws IOException {
         RestUtils.get(
             authToken,
             collectionPath
-                + "/cool-search-id?where={\"quiz.nests.q2.options.*.this.that.them\": {\"$eq\": false}}&raw=true",
+                + "/cool-search-id?where={\"quiz.nests.q2.options.[*].this.that.them\": {\"$eq\": false}}&raw=true",
```

Comment:
> Should we rather allow it, basically * would match path or array, [*] would match array only?

I think this ^ makes sense.

Reply:
Allow it? So revert the test and fix the matching on `master`?

Reply:
Agreed star matches everything.

Reply:
Done in #1116